sdk/core/azure-core/azure/core/pipeline/policies/_retry.py

```diff
@@ -45,11 +45,12 @@
 
 from ._base import HTTPPolicy, RequestHistory
 from . import _utils
-
+from ..._enum_meta import CaseInsensitiveEnumMeta
 
 _LOGGER = logging.getLogger(__name__)
 
-class RetryMode(str, Enum):
+class RetryMode(str, Enum, metaclass=CaseInsensitiveEnumMeta):
+    # pylint: disable=enum-must-be-uppercase
     Exponential = 'exponential'
     Fixed = 'fixed'
```

Comment:
Why is this disabled if we are in fact following the rule?

Reply:
I wasn't sure if this would be a breaking change to make it EXPONENTIAL and FIXED

Reply:
Ooooh, it's saying the entire thing should be all caps. Okay, yeah that would be. I presume the case insensitivity only applies to the string value. If it actually applied to the attribute label I'd be surprised (and impressed honestly)